c10/util/BFloat16-inl.h

```diff
@@ -263,5 +263,9 @@ class numeric_limits<c10::BFloat16> {
 /// Used by vec256<c10::BFloat16>::map
 inline c10::BFloat16 exp(c10::BFloat16 a) { return std::exp(float(a)); }
 inline c10::BFloat16 log(c10::BFloat16 a) { return std::log(float(a)); }
-
+inline c10::BFloat16 abs(c10::BFloat16 a) { return std::abs(float(a));}
+inline c10::BFloat16 tanh(c10::BFloat16 a) { return std::tanh(float(a));}
+inline c10::BFloat16 min(c10::BFloat16 a, c10::BFloat16 b) { return std::min(float(a), float(b));}
+inline c10::BFloat16 max(c10::BFloat16 a, c10::BFloat16 b) { return std::max(float(a), float(b));}
```

Comment:
why cant we do something like return `(b > a) ? b : a;` and avoid casting?

Reply:
BF16 does not supported compare operators. Therefore, we need to cast BF16 to FP32 and then compare it. In addtion, we try to align PyTorch code rule because almost all PyTorch code calls std::min/std::max to get minimum/maxmum data.